seed/static/seed/js/seed.js

```diff
@@ -1064,6 +1065,33 @@ SEED_app.config(['stateHelperProvider', '$urlRouterProvider', '$locationProvider
           }]
         }
       })
+      .state({
```

Comment:
This was essentially copied from the 'inventory_list' state. I left a lot of the `resolve` components, as I expected them to be used later for filtering.